jetstream/analysis.py

```diff
@@ -216,22 +238,30 @@ def calculate_statistics(
         """
         Run statistics on metric.
         """
-        return metric.run(segment_data, self.config.experiment).set_segment(segment)
+        return (
+            metric.run(segment_data, self.config.experiment)
+            .set_segment(segment)
+            .set_analysis_basis(metric.metric.analysis_basis)
+        )
 
     @dask.delayed
-    def counts(self, segment_data: DataFrame, segment: str) -> StatisticResultCollection:
+    def counts(
+        self, segment_data: DataFrame, segment: str, analysis_basis: AnalysisBasis
+    ) -> StatisticResultCollection:
         """Count and missing count statistics."""
+        metric = analysis_basis.value + "_identity"
```

Comment:
For `counts`, this will take the analysis basis into account. So if there are metrics that are based on enrollments and some based on exposures, then the statistics result table will have `enrollments_identity` and `exposures_identity` metrics for each branch

Reply:
this is a breaking change for the console so we need to figure out how to handle that before merging (maybe revisiting the adr)

Reply:
actually, renaming is not necessary anymore now that we have an analysis_basis column